Makefile

```diff
@@ -48,7 +48,7 @@ copy-android-bindings:
 
 .PHONY: copy-ios-bindings
 copy-ios-bindings:
-	@cp -R cmd/wallet-sdk-gomobile/bindings/ios/walletsdk.xcframework demo/app/ios/Runner
+	@rm -rf demo/app/ios/Runner/walletsdk.xcframework && cp -R cmd/wallet-sdk-gomobile/bindings/ios/walletsdk.xcframework demo/app/ios/Runner
```

Comment:
Reason for this change?

Reply:
When previously generate locally, running this target the second time would throw an error since it can't override and existing folder. This might have to do with the local default permissions, but  I think cleaning the old binding doesn't hurt.